### PR TITLE
rsh: add page

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/linux/rsh.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# rsh
+
+> Execute commands on a remote host.
+> More information: <https://www.gnu.org/software/inetutils/manual/html_node/rsh-invocation.html>.
+
+- Execute a command on a remote host:
+
+`rsh {{remote_host}} ls -l`
```
Comment (gutjuri) [inline pages/linux/rsh.md:8]:
```suggestion
`rsh {{remote_host}} {{ls -l}}`
```

File: pages/linux/rsh.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Execute a command on a remote host with a specific username:
+
+`rsh {{remote_host}} -l {{username}} ls -l`
```
Comment (gutjuri) [inline pages/linux/rsh.md:12]:
```suggestion
`rsh {{remote_host}} -l {{username}} {{ls -l}}`
```

File: pages/linux/rsh.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Redirect `stdin` to `/dev/null` when executing a command on a remote host:
+
+`rsh {{remote_host}} --no-err ls -l`
```
Comment (gutjuri) [inline pages/linux/rsh.md:16]:
```suggestion
`rsh {{remote_host}} --no-err {{ls -l}}`
```

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thanks, I have three suggestions. `ls -l` is supposedly a placeholder that can stand for any command?

**Review by gutjuri — APPROVED** [review]

**Pull request merged**